tests/omhttp-basic.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# This file is part of the rsyslog project, released under ASL 2.0
+port=8080
+omhttp_start_server -p $port
+
+#  Starting actual testbench
+. ${srcdir:=.}/diag.sh init
```

Comment:
This needs to be the first thing the testbench does. As far as I can see, this test should also fail because `omhttp_start_server` is unknown at the time it is called.

Reply:
Moved this to be the first thing each omhttp test does.